src/js/components/List/List.js

```diff
@@ -249,54 +256,60 @@ const List = React.forwardRef(
+                : Math.max(active - 2, 1),
+            );
+          } else if (
+            disabledItems?.includes(getValue(data[active], active, itemKey))
+          ) {
+            event.preventDefault();
+          } else if (onClickItem) {
+            event.persist();
+            const adjustedEvent = event;
+            adjustedEvent.item = data[active];
+            adjustedEvent.index = active;
+            onClickItem(adjustedEvent);
+            sendAnalytics({
+              type: 'listItemClick',
+              element: listRef.current,
+              event: adjustedEvent,
+              item: data[active],
+              index: active,
+            });
+          }
+        }
+      },
+      [
+        active,
+        onOrder,
+        orderableData,
+        pinnedInfo,
+        disabledItems,
+        data,
+        itemKey,
+        onClickItem,
+        updateActive,
+      ],
+    );
+
     return (
       <Container {...containterProps}>
         <Keyboard
-          onEnter={
-            (onClickItem || onOrder) && active >= 0
-              ? (event) => {
-                  if (onOrder) {
-                    const index = Math.trunc(active / 2);
-                    // Call onOrder with the re-ordered data.
-                    // Update the active control index so that the
-                    // active control will stay on the same item
-                    // even though it moved up or down.
-                    if (active % 2) {
-                      onOrder(
-                        reorder(orderableData, pinnedInfo, index, index + 1),
-                      );
-                      updateActive(
-                        Math.min(active + 2, orderableData.length * 2 - 2),
-                      );
-                    } else {
-                      onOrder(
-                        reorder(orderableData, pinnedInfo, index, index - 1),
-                      );
-                      updateActive(Math.max(active - 2, 1));
-                    }
-                  } else if (
-                    disabledItems?.includes(
-                      getValue(data[active], active, itemKey),
-                    )
-                  ) {
-                    event.preventDefault();
-                  } else if (onClickItem) {
-                    event.persist();
-                    const adjustedEvent = event;
-                    adjustedEvent.item = data[active];
-                    adjustedEvent.index = active;
-                    onClickItem(adjustedEvent);
-                    sendAnalytics({
-                      type: 'listItemClick',
-                      element: listRef.current,
-                      event: adjustedEvent,
-                      item: data[active],
-                      index: active,
-                    });
-                  }
-                }
-              : undefined
-          }
+          onEnter={onSelectOption}
+          onSpace={onSelectOption}
```

Comment:
Can we add `event.preventDefault` to `onSpace` so that the screen doesn't scroll down when we hit space? It also might be nice to add `event.preventDefault` to `onUp` and `onDown`, I realize that is going outside of the scope of the original issue though so I will leave it up to you if you want to include that part or not.
```suggestion
          onSpace={(event) => {
            event.preventDefault();
            onSelectOption(event);
          }}
```

Reply:
hey @jcfilben  for now, added event.preventDefault to `onSpace`
`onUp` and `onDown` will raise a different PR